mne/viz/evoked.py

```diff
@@ -269,8 +269,7 @@ def _plot_evoked(evoked, picks, exclude, unit, show, ylim, proj, xlim, hline,
                     units, scalings, hline, gfp, types, zorder, xlim, ylim,
                     times, bad_ch_idx, titles, ch_types_used, selectable,
                     False, line_alpha=1.)
-        for ax in axes:
-            ax.set_xlabel('Time (ms)')
+        plt.setp(axes, xlabel='Time (ms)')
```

Comment:
you want to change all these to 'Time (s)' right?

Reply:
In #3164 this is discussed, yes. It's not implemented here, though, and would be better as a follow-up PR if people agree we should do it.